src/components/pages/get-a-quote/form.svelte

```diff
@@ -133,10 +133,10 @@
             {@html eggs[character]}
             {#if character === 't-shirt' && truncatedText}
               <div
-                class="absolute left-[235px] top-[40px] line-clamp-2 flex h-[76px] w-36 items-center justify-center text-center font-comic font-bold leading-snug"
+                class="absolute left-[235px] top-[40px] text-black line-clamp-2 flex h-[76px] w-36 items-center justify-center text-center font-comic font-bold leading-snug"
```

Comment:
avoid using `black`

Reply:
That shouldn't be there, it isn't even related to my code, thank you for noticing 🙌